### PR TITLE
remove dataportaal specific implementation of the paragraph

### PR DESCRIPTION
**Pull request by aditudorache**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/Typography/TypographyStyle.ts [hunk 1]
```diff
@@ -31,12 +28,6 @@ export default styled.p<Props>`
   ${getTypographyFromTheme()};
   font-stretch: normal;
   letter-spacing: normal;
-  ${({ strong }) =>
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Typography/TypographyStyle.ts]: please revert this. Now we're only able to set strong on paragraph and no other typeography-related components

File: packages/asc-ui/src/components/Paragraph/ParagraphStyle.ts [hunk 2]
```diff
@@ -17,14 +17,6 @@ export const ParagraphStyleCSS = css<Props>`
       font-weight: 700;
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Paragraph/ParagraphStyle.ts]: Please do remove this strong prop here, as we already use it in typography

File: packages/asc-ui/src/components/Typography/TypographyStyle.ts (hunk 1 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Typography/TypographyStyle.ts]: Sure! Thanks!

File: packages/asc-ui/src/components/Paragraph/ParagraphStyle.ts (hunk 2 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Paragraph/ParagraphStyle.ts]: done

**Pull request merged**